Sources/Datadog/RUM/AutoInstrumentation/Actions/UIKitRUMUserActionsHandler.swift

```diff
@@ -46,14 +48,16 @@ internal class UIKitRUMUserActionsHandler: UIKitRUMUserActionsHandlerType {
             )
         }
 
-        subscriber?.process(
-            command: RUMAddUserActionCommand(
-                time: dateProvider.currentDate(),
-                attributes: [:],
-                actionType: .tap,
-                name: targetName(for: actionTargetView)
+        if let rumAction = userActionsPredicate.rumAction(targetView: actionTargetView) {
+            subscriber?.process(
```

Comment:
nit-picking: if we first nil-check `subscriber`, we can avoid unnecessary calls to `userActionsPredicate`
```suggestion
        if let subscriber = subscriber,
           let rumAction = userActionsPredicate.rumAction(targetView: actionTargetView) {
            subscriber.process(
```

Reply:
Indeed, thanks for the suggestion. I've added early return if `subscriber` is not set.